builder/amazon/ebs/step_stop_instance.go

```diff
@@ -23,16 +24,20 @@ func (s *stepStopInstance) Run(state multistep.StateBag) multistep.StepAction {
 		return multistep.ActionContinue
 	}
 
-	// Stop the instance so we can create an AMI from it
-	ui.Say("Stopping the source instance...")
-	_, err := ec2conn.StopInstances(&ec2.StopInstancesInput{
-		InstanceIds: []*string{instance.InstanceId},
-	})
-	if err != nil {
-		err := fmt.Errorf("Error stopping instance: %s", err)
-		state.Put("error", err)
-		ui.Error(err.Error())
-		return multistep.ActionHalt
+	var err error
+
+	if s.DisableStopInstance == false {
```

Comment:
Style: this should be written as `if !s.DisableStopInstance`

I think we  should also have an `else` block here that sends a message to the UI indicating that the instance will continue to run and packer will continue to wait until the user intervenes.